resources/scponly_user.rb

```diff
@@ -0,0 +1,124 @@
+# true To learn more about Custom Resources, see https://docs.chef.io/custom_resources.html
+
+resource_name :scponly_user
+
+default_action :create
+
+property :write_dir, String, default: 'incoming'
+property :public_key, String
+property :private_key, String
+property :chroot, [true, false], default: true
+property :altroot, String, default: '/var/lib/chroots'
```

Comment:
I wanted this to use the resource name as a sub directory. 
```suggestion
property :altroot, String, default: "/var/lib/chroots/#{new_resource.name}"
```